src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball();
+    }
```

Comment:
randomize color and number in this method
for color - use ColorSupplier instance and randomize the color